### ISSUE TITLE
Debug information causes server shutdown process to halt

### ISSUE THREAD

**Issue by KelvinZhang512** [Builderb0y/BigGlobe#81]
My server shutdown process was stuck every time when this message pops out:
ColumnEntryRegistry resetting: builderb0y.bigglobe.columns.scripted.ColumnEntryRegistry$Loading@47ecf003; override: null
[latest.log](https://github.com/user-attachments/files/18218689/latest.log)

**Comment by Builderb0y** [comment]
Can you try to make a thread dump with visualVM? That debug message doesn't loop or wait on anything, so I doubt it's the cause. More likely it just happened shortly before the server got stuck.

**Comment by KelvinZhang512** [comment]
I'm afraid it's unable to do so, because it seems that my server is using an unsupported jvm, but I'll try to bring some extra debug info.

**Comment by Builderb0y** [comment]
Make sure the server and visualVM are both running with the JDK, not the JRE. You may need to install an additional version of java for this. [Adoptium](https://adoptium.net/temurin/releases/?package=jdk) should work.

**Comment by Builderb0y** [comment]
Any updates?

**Comment by KelvinZhang512** [comment]
No, it seems that I'm unable to use visualVM to connect to my server, but I can solve the halt by adding a detector and force it to shutdown.

**Comment by Builderb0y** [comment]
I think I'll close this issue for now. Feel free to comment again if you do manage to obtain more information.
